### PR TITLE
LPS-84130 Add a property to allow fallback

### PR DESCRIPTION
**Pull request by dantewang**

### CODE REVIEW

**Comment by dantewang** [comment]
ci:test

I also ran CI with this property set to true yesterday, and the result had no unique failure, see https://test-1-18.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/306/jenkins-report.html

**Comment by dantewang** [comment]
cc @emmaliu

File: portal-impl/src/com/liferay/portal/util/PropsValues.java [hunk 1]
```diff
@@ -1574,6 +1574,8 @@
 	@Deprecated
 	public static final int SEARCH_CONTAINER_SHOW_PAGINATION_TOP_DELTA = GetterUtil.getInteger(PropsUtil.get(PropsKeys.SEARCH_CONTAINER_SHOW_PAGINATION_TOP_DELTA), 10);
 
+	public static final boolean SERVLET_CONTEXT_CLASS_LOADER_POOL_FALLBACK = GetterUtil.getBoolean(PropsUtil.get(PropsKeys.SERVLET_CONTEXT_CLASS_LOADER_POOL_FALLBACK));
```
Comment (tinatian) [inline portal-impl/src/com/liferay/portal/util/PropsValues.java:1577]: @dantewang why are you adding this? I did not see the usage of it.

Tina

Comment (dantewang) [reply portal-impl/src/com/liferay/portal/util/PropsValues.java:1577]: @tinatian There's no usage of it. I thought `PropsKeys` and `PropsValues` need to be consistent, like `UNICODE_TEXT_NORMALIZER_FORM`. If not, I'll resend without this.

Comment (tinatian) [reply portal-impl/src/com/liferay/portal/util/PropsValues.java:1577]: @dantewang 

We only add it when it is used, it does not make sense to  me you add it to PropsValues, and use PropsKeys directly in your code. 

Either you add it to PropsValues, and use PropsValues in your code.
Or you add it only to PropsKeys, and use PropsKeys directly in your code.

I can accept both.

Tina.

**Comment by tinatian** [comment]
@dantewang See my comment and please resend.

Thanks.
Tina.

**Pull request closed without merging**